### ISSUE TITLE
[2.8.0] pre-built binary releases for Linux and Mac report version as UNKNOWN

### ISSUE THREAD

**Issue by robert914** [hadolint/hadolint#730]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

The `-v`/`--version` output reports `UNKNOWN` for the pre-compiled release binaries distributed here: https://github.com/hadolint/hadolint/releases/tag/v2.8.0

### Expected behavior
On Mac:
```bash
❯ ./hadolint-Darwin-x86_64 -v
Haskell Dockerfile Linter 2.8.0-no-git
```
On Linux:
```bash
❯ ./hadolint-Linux-x86_64 -v
Haskell Dockerfile Linter 2.8.0-no-git
```

### Actual behavior
On Mac:
```bash
❯ ./hadolint-Darwin-x86_64 -v
Haskell Dockerfile Linter UNKNOWN
```
On Linux:
```bash
❯ ./hadolint-Linux-x86_64 -v
Haskell Dockerfile Linter UNKNOWN
```

### Steps to reproduce the behavior

See above.

**Dockerfile** (if relevant)

N/A

**Additional environment details (OS, stack version, etc.)**

Mac or Linux OS versions, possibly Windows (untested).

**Comment by lorenzo** [comment]
no idea why, must be some change in the CI which I'm unaware of. Help appreciated

**Comment by dmpe** [comment]
Jop, can also confirm that.

**Comment by mjsteinbaugh** [comment]
I'm seeing this on macOS with Homebrew binary for v2.8.0.

**Comment by darkvertex** [comment]
The Windows binary has this issue too.
